core/app/workers/workarea/update_email.rb

```diff
@@ -0,0 +1,33 @@
+module Workarea
+  class UpdateEmail
```

Comment:
I checked, this wasn't being decorated anywhere I could find and I wanted to keep the number of Sidekiq jobs at a minimum.